### DISCUSSION TITLE
Injected ts-jest transforms

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [nrwl/nx#13926] [General]
I just stumbled upon ts-jest injecting a transform into the jest config as it seems. This can lead to hideous errors when you add the transform yourself, but do not have the exact same regex as the injected transform.
You might end up with 2 transforms for the same files and the last one wins, and that is always the injected one.
For example, if you add a path to the tsconfig.spec.json in your custom transform, that will be ignored if the regex does not match, resulting in ts miraculously not finding some type definitions anymore, while vscode finds them perfect.

**Comment by qxb3** [comment]
kekw

**Comment by SamirPS** [comment]
What is this ?
